compare only the first 5 chars of zlibVersion and ZLIB_VERSION since the former
gives 1.3.3, while the later 1.3.3.2
---
 src/libs/luazlib/lzlib.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: luatex-0.10.1/src/libs/luazlib/lzlib.c
===================================================================
--- luatex-0.10.1.orig/src/libs/luazlib/lzlib.c	2007-08-06 17:38:12.000000000 +0200
+++ luatex-0.10.1/src/libs/luazlib/lzlib.c	2007-08-06 17:38:33.000000000 +0200
@@ -552,7 +552,7 @@
 
     /* make sure header and library version are consistent */
     const char* version = zlibVersion();
-    if (strcmp(version, ZLIB_VERSION))
+    if (strncmp(version, ZLIB_VERSION, 5))
     {
         lua_pushfstring(L, "zlib library version does not match - header: %s, library: %s", ZLIB_VERSION, version);
         lua_error(L);
